gammapy/modeling/models/spatial.py

```diff
@@ -107,6 +149,11 @@ def evaluate_geom(self, geom):
         w = self._grid_weights(x, y, x0, y0)
         return w / geom.solid_angle()
 
+    @property
+    def to_region(self):
```

Comment:
It's weird to have `model.to_region` as a property. It should be either `model.to_region()` (method call), or `model.region` (property).

My suggestion would be to change everywhere to this and remove the `@property`
```
def to_region(self, **kwargs):
    return SomeRegion(..., ***kwargs)
```
Passing along kwargs to the sky region init means that callers can e.g. set colors or styles to make it pretty how they like.
Methods are also the more flexible way, in case we want to add arguments to the API later, we can without breaking all callers. With properties you have to be sure that you'll never want arguments.